api/types/user_fields.go

```diff
@@ -22,6 +22,7 @@ import (
 )
 
 // SignupFields is a set of fields that use to sign up to yorkie server.
+// // must be same with [ChangePasswordFields].
 type SignupFields struct {
```

Comment:
How about changing `SignupFields` to `UserFields` just like file name you changed?

Reply:
That's a really good idea :)